proxy-manager/DOCS.md

```diff
@@ -79,6 +78,9 @@ cleared automatically_
   contain the data from this add-on. Please make sure you backup both
   the Nginx Proxy Manager and the MariaDB add-ons.
 
+
+**Note**: Since Home Assistant 2021.7 connections from proxies are refused with HTTP code 400. In case you want to proxy to `http://homeassistant:8123` a rule must be added to configuration [as described here][reverseproxies].
```

Comment:
This is unrelated and incorrect as well. Proxies always had to be configured in Home Assistant.